.github/workflows/protobufs.yml

```diff
@@ -1,84 +0,0 @@
-name: Protobufs
-
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-  schedule:
-    - cron: "0 0 * * *"
-
-jobs:
-  validate_protobufs:
```

Comment:
The `validate_protobufs` step is still useful here.  This catches all general syntax errors in *.textproto files and *.proto files.  This isn't specific to feature.textproto validations.

I think this file should remain with everything inside the`validate_oc_paths` section removed.

Reply:
Thanks!  Done.

Reply:
Done